stdlib/messaging/rabbitmq/src/main/java/org/ballerinalang/messaging/rabbitmq/util/ConnectionUtils.java

```diff
@@ -83,6 +108,84 @@ public static Connection createConnection(MapValue<String, Object> connectionCon
         }
     }
 
+    /**
+     * Creates and retrieves the initialized SSLContext.
+     *
+     * @param secureSocket secureSocket record.
+     * @return Initialized SSLContext.
+     */
+    private static SSLContext getSSLContext(MapValue secureSocket) {
+        final String CERT_PASS = "password";
+        final String CERT_PATH = "path";
+        try {
+            MapValue cryptoKeyStore = secureSocket.getMapValue(RabbitMQConstants.RABBITMQ_CONNECTION_KEYSTORE);
+            MapValue cryptoTrustStore = secureSocket.getMapValue(RabbitMQConstants.RABBITMQ_CONNECTION_TRUSTORE);
+
+            char[] keyPassphrase = cryptoKeyStore.getStringValue(CERT_PASS).toCharArray();
+            String keyFilePath = cryptoKeyStore.getStringValue(CERT_PATH);
+
+            char[] trustPassphrase = cryptoTrustStore.getStringValue(CERT_PASS).toCharArray();
+            String trustFilePath = cryptoTrustStore.getStringValue(CERT_PATH);
+
+            String tlsVersion = secureSocket.getStringValue(RabbitMQConstants.RABBITMQ_CONNECTION_TLS_VERSION);
+            if (tlsVersion == null) {
+                tlsVersion = "TLS";
+            }
+
+            KeyStore keyStore = KeyStore.getInstance(RabbitMQConstants.KEY_STORE_TYPE);
+            if (keyFilePath != null) {
+                keyStore.load(new FileInputStream(keyFilePath), keyPassphrase);
+            } else {
+                throw new RabbitMQConnectorException(RabbitMQConstants.CREATE_SECURE_CONNECTION_ERROR +
+                        "Path for the keystore is not found.");
+            }
+            KeyManagerFactory keyManagerFactory =
+                    KeyManagerFactory.getInstance(KeyManagerFactory.getDefaultAlgorithm());
+            keyManagerFactory.init(keyStore, keyPassphrase);
+
+            KeyStore trustStore = KeyStore.getInstance(RabbitMQConstants.KEY_STORE_TYPE);
+            if (trustFilePath != null) {
+                trustStore.load(new FileInputStream(trustFilePath), trustPassphrase);
+            } else {
+                throw new RabbitMQConnectorException("Path for the truststore is not found.");
```

Comment:
Shall we include the provided path as well? That will be helpful to identify the error.

Reply:
trustFilePath here is null.